client/js/screens/LogIn/LogIn.js

```diff
@@ -1,9 +1,81 @@
 import React from "react";
-import { Text } from "react-native";
+import { Form, Field } from "react-final-form";
+import {
+  ImageBackground,
+  View,
+  Image,
+  TouchableOpacity,
+  TextInput,
+  Text,
+  StatusBar
+} from "react-native";
 import styles from "./styles";
 
-const Login = () => {
-    return <Text>I am the Login Screen</Text>;
+const showResults =  async (values, login) => {
+  const res = await login({variables: values})
+  console.log("res",res)
+};
+
+const Login = ({ login }) => {
+  return (
+    <View style={styles.loginPage}>
+      <StatusBar barStyle="light-content" />
+      <ImageBackground
+        source={require("./background.png")}
+        style={{
+          width: "100%",
+          height: "100%"
```

Comment:
use styles.js file for all of your styles and use resize mode or Dimensions or flexbox. Try avoiding height and width 100%.